app/assets/javascripts/discourse/controllers/edit-category.js.es6

```diff
@@ -101,6 +102,18 @@ export default Ember.Controller.extend(ModalFunctionality, {
           self.set('deleting', false);
         }
       });
+    },
+
+    showDeleteTooltip() {
+      // check if is touch device
+      if ($('html').hasClass('discourse-touch')) {
+
+        if(this.get('hiddenTooltip')){
+          this.set('hiddenTooltip', false);
+        } else {
+          this.set('hiddenTooltip', true);
```

Comment:
@SamSaffron I improve the actions as you suggest. I notice my code wasn't following the rest but i needed little guide on how to improve it. So thank you for the hint.
Let me know if now looks better